nipoppy/trackers/run_tracker.py

```diff
@@ -34,6 +32,13 @@
     "mriqc": mriqc_tracker.tracker_configs,
     "tractoflow": tractoflow_tracker.tracker_configs,
 }
+PIPELINE_REQUIRED_DATATYPES = {
+    "heudiconv": [],
+    "freesurfer": ["anat"],
+    "fmriprep": ["anat"],
+    "mriqc": ["anat"],
+    "tractoflow": ["anat", "dwi"],
+}
```

Comment:
Should this go in global_config?

Reply:
I was thinking this should go in the individual trackers instead? It would be the datatypes required for the `pipeline_complete` part of the pipeline to run

Reply:
Individual trackers makes sense!